### PR TITLE
More additions to the 1.13 Release Team

### PR DESCRIPTION
**Pull request by justaugustus**
Just a note that these additions are subject to change. We're still shuffling a few people around.

/sig release
/assign @AishSundar 
/hold

Signed-off-by: Stephen Augustus <foo@agst.us>

### CODE REVIEW

File: releases/release-1.13/release_team.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 | **Role** | **Name** (**GitHub / Slack ID**)  | **Shadow Name(s) (GitHub / Slack ID)** |
 | ------ | ------ | ------ |
 | Lead | Aishwarya Sundar ([@AishSundar](https://github.com/AishSundar)) | Aaron Crickenberger ([@spiffxp](https://github.com/spiffxp)) |
-| Enhancements | Kendrick Coleman ([@kacole2](https://github.com/kacole2)) | Michael Singh ([@wadadli](https://github.com/wadadli)), Guinevere Saenger ([@guineveresaenger](https://github.com/guineveresaenger) / @gsaenger on Slack), Arnaud Meukam ([@ameukam](https://github.com/ameukam)) |
+| Enhancements | Kendrick Coleman ([@kacole2](https://github.com/kacole2)) | Michael Singh ([@wadadli](https://github.com/wadadli)), Guinevere Saenger ([@guineveresaenger](https://github.com/guineveresaenger) / @gsaenger on Slack), Arnaud Meukam ([@ameukam](https://github.com/ameukam)), Claire Laurence ([@claurence](https://github.com/claurence)) |
 | CI Signal | Josh Berkus ([@jberkus](https://github.com/jberkus)) | Shivaani Mall ([@shivani1494](https://github.com/shivani1494)), Morten Torkildsen ([@mortent](https://github.com/mortent)) |
```
Comment (AishSundar) [inline releases/release-1.13/release_team.md]: From the GH issue, can we please add TaoBeier (@tao12345666333) and Maria Ntalla (@mariantalla) as shadows for CI Signal?

File: releases/release-1.13/release_team.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 | Test Infra | Cole Wagner ([@cjwagner](https://github.com/cjwagner)) | Amit Watve ([@amwat](https://github.com/amwat)), Hannes Hoerl ([@hoegaarden](https://github.com/hoegaarden)) |
```
Comment (hoegaarden) [inline releases/release-1.13/release_team.md]: As discussed with @AishSundar I will step down as test-infra shadow and **only** shadow Doug on the Release Branch Management.

**Comment by AishSundar** [comment]
Thanks a lot @justaugustus 

/lgtm

**Pull request merged**